doc-content/enterprise-only/project-data/preview-editor-copy-test-proc.adoc

```diff
@@ -0,0 +1,14 @@
+[id='preview-editor-copy-test-proc']
+= Copying a test scenario
+
+You can copy an existing test scenario to the same package or to some other package by using the *Copy* button from the upper-right toolbar.
+
+.Procedure
+. From the *Test Scenario (Preview)* editor toolbar on the upper-right, click *Copy*.
+. In the *Make a Copy* window,
+* Enter a name in the *New Name* field.
```

Comment:
These four points are steps, hence need to be numbered and not presented as a bullet. So we can change these 4 to sub-steps under the step 2.

Reply:
1. Changed as suggested.
2. No change as "Click *Make a Copy*." occurs or happens in the *Make a Copy* window.
3. Changed as suggested.